scripts/rewards/DistributeRewards.ts

```diff
@@ -0,0 +1,21 @@
+// TODO fill vars
+import { ethers } from 'hardhat'
+import { parseEther } from "ethers/lib/utils";
+const sPERC = "";
+const sLPPERC = "";
+const dailySPERCAllocation = parseEther("1357.4");
+const dailyLPAllocation = parseEther("5429.6");
```

Comment:
Just curious, these are arbitrary values, right?
Or is it like a max amount that can be distributed on a single day?

Reply:
No, they are the actual daily drop amounts in PERC for each staking pool.

Reply:
Got it.

Actually, I don't haha :sweat_smile: 

My understanding of the contract is very lacking.
I'll take some moments to study it's mechanics a little more over the next days.

Thanks man!